qlib/data/dataset/processor.py

```diff
@@ -102,6 +102,25 @@ def __call__(self, df):
         return df.loc[:, ~mask]
 
 
+class FilterCol(Processor):
+    def __init__(self, fields_group="feature", col_list=[]):
+        self.fields_group = fields_group
+        self.col_list = col_list
+
+    def __call__(self, df):
+
+        cols = get_group_columns(df, self.fields_group)
+        all_cols = df.columns
+        diff_cols = np.setdiff1d(all_cols.get_level_values(-1), cols.get_level_values(-1))
+        self.col_list = np.union1d(diff_cols, self.col_list)
+
+        if isinstance(df.columns, pd.MultiIndex):
```

Comment:
@lwwang1995  Why can't we use `df.columns.get_level_values(-1).isin(self.col_list)` directly? 
get_level_values also support single index.
```python

In [95]: df
Out[95]:
                0         1         2         3
bar one  1.807612  1.036448  1.113804 -1.736091
    two  0.684449 -0.675280 -0.442803  1.186755
baz one -0.161282 -1.272930  0.709273 -0.563070
    two  1.565204 -0.146586 -0.975392 -0.088376
foo one  1.571692  0.062846 -1.039941  0.057195
    two -0.089261 -0.191429 -0.834761 -1.028378
qux one -0.105965 -0.577452 -0.466957 -0.357573
    two  1.236167 -1.231718 -0.425552  0.795854

In [96]: df.columns.get_level_values(-1).isin([1])
Out[96]: array([False,  True, False, False])
```